Octokit/Models/Response/PullRequest.cs

```diff
@@ -219,6 +220,11 @@ public bool Merged
         /// </summary>
         public bool Locked { get; protected set; }
 
+        /// <summary>
+        /// If users with write access to the base repository can push to the HEAD branch
+        /// </summary>
+        public bool MaintainerCanModify { get; protected set; }
```

Comment:
I think we should make this nullable `bool? MaintainerCanModify` because responses that return lists of pull requests will not return this field, so rather than indicate it as `false` in that case, we should indicate it as `null`